config/initializers/rswag_ui.rb

```diff
@@ -5,7 +5,9 @@
   # NOTE: If you're using rspec-api to expose Swagger files (under swagger_root) as JSON or YAML endpoints,
   # then the list below should correspond to the relative paths for those endpoints
 
-  c.swagger_endpoint "/api-docs/v1/swagger.yaml", "API V1 Docs"
+  # TODO: testing, will need to use `Settings.environment.eql?("non-live")` to differentiate uat/staging from actual "production"
+  c.swagger_endpoint "/api-docs/v1/live/swagger.yaml", "API V1 (live) Docs"
+  c.swagger_endpoint "/api-docs/v1/swagger.yaml", "API V1 (non-live) Docs" unless Rails.env.production?
```

Comment:
can't actually test on live/production so if happy with this approach then we can just switch this conditional out to use `Settings.environment.eql?("non-live")`. and the smoke test endpoints should be hidden on production but not staging and UAT.

Reply:
Could it be something like...
```suggestion
  if Settings.environment.eql?("non-live")
    c.swagger_endpoint "/api-docs/v1/swagger.yaml", "API V1 Docs" 
  else
    c.swagger_endpoint "/api-docs/v1/live/swagger.yaml", "API V1 Docs"
  end
```
to make it less obvious that we have two versions?

Reply:
No, good shout - maybe use non-live as a label, so production will just be `API V1 Docs` and non-live is  `API V1 (inc smoke-tests) Docs` ?